Mathlib/Topology/CompactOpen.lean

```diff
@@ -109,6 +110,17 @@ theorem continuous_comp_left (f : C(X, Y)) : Continuous (fun g => g.comp f : C(Y
     simpa only [mapsTo_image_iff] using isOpen_setOf_mapsTo (hK.image f.2) hU
 #align continuous_map.continuous_comp_left ContinuousMap.continuous_comp_left
 
+/-- Any pair of homeomorphisms `X ≃ₜ Z` and `Y ≃ₜ T` gives rise to a homeomorphism
+`C(X, Y) ≃ₜ C(Z, T)`. -/
+protected def _root_.Homeomorph.continuousMapCongr (φ : X ≃ₜ Z) (ψ : Y ≃ₜ T) :
```

Comment:
Looking through the library I think this is the standard name. 
```suggestion
protected def _root_.Homeomorph.arrowCongr (φ : X ≃ₜ Z) (ψ : Y ≃ₜ T) :
```

Reply:
I was going to use it but figured it would sound a bit weird for the uniform version since `ContinuousMap` are not the arrows in the category of uniform spaces. But oh well, consistency is more important.